src/common/SurgePatch.cpp

```diff
@@ -741,7 +740,7 @@ void SurgePatch::load_patch(const void* data, int datasize, bool preset)
    patch_header* ph = (patch_header*)data;
    ph->xmlsize = vt_read_int32LE(ph->xmlsize);
 
-   if ((int)ph->tag == vt_read_int32BE('sub3'))
+   if (!memcmp(&ph->tag, "3bus", 4))
```

Comment:
After taking all the endian-mangling into account, I believe this actually should be "sub3".